app/models/repository.rb

```diff
@@ -4,7 +4,7 @@
 class Repository < Sequel::Model
   SLUG_BLACKLIST = %w(repositories organizations members settings).freeze
   CONTENT_TYPES = %w(ontology model specification mathematical).freeze
-  REMOTE_TYPES = %(fork mirror).freeze
+  REMOTE_TYPES = %(fork mirror)
```

Comment:
I think this should be `%w(fork mirror).freeze` instead. `%(fork mirror)` is just `"fork mirror"`.

Reply:
You're right. Fixed in fb5b53d